charts/graph-node/values.yaml

```
@@ -0,0 +1,46 @@
+serviceMonitor:
```

Can add a global values and template them down the chart
```suggestion
global:
  namespaceOverride: ""
  replicaCount: 1

serviceMonitor:
```